examples/nextjs/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "@contentful/live-preview": "^2.0.2",
+    "next": "latest",
```

Comment:
Is this always guaranteed to work with latest? might be safer to select a version just in case there are any breaking changes in next